.maint/ci/activate.sh

```diff
@@ -0,0 +1,9 @@
+if [ -e virtenv/bin/activate ]; then
```

Comment:
I would strongly advise switching to tox instead. I just moved pybids from these kinds of scripts.

Reply:
I gave it a shot. Thanks!